### DISCUSSION TITLE
Feature request: Now playing embed

### DISCUSSION THREAD

**Discussion by aftechro** [AzuraCast/AzuraCast#6996] [Ideas]
Hey guys,

To make it simple for non-devs, instead using the api as i've seen in the discussions section, for the now playing, can you create the embed iframe that can be easily integrated with another website?

![image](https://github.com/AzuraCast/AzuraCast/assets/38830718/be0822c0-314c-4d63-8901-3f491e306831)

![image](https://github.com/AzuraCast/AzuraCast/assets/38830718/3eec03c1-1b1f-4881-96cf-6809063694c8)

Thank you!

**Comment by BusterNeece** [comment]
Hello! The best place to suggest new features would be our FeatureUpvote community here: https://features.azuracast.com/
